src/base/users/View.svelte

```diff
@@ -161,96 +161,96 @@
-                    <Project {project} user={addressOrName} config={profile.config(config)}>
-                      <span slot="stateHash">
-                        <span class="mobile">commit {formatCommit(project.anchor.stateHash)}</span>
-                        <span class="desktop">commit {project.anchor.stateHash}</span>
-                      </span>
-                    </Project>
-                  </div>
-                {/each}
-              {:catch err}
-                <Message error>
-                  <strong>Error: </strong> failed to load projects: {err.message}.
-                </Message>
-              {/await}
-            {/if}
-          {/await}
+      <div class="desktop">
+        {#if isAuthorized(profile.address)}
+          <button class="tiny secondary" on:click={setName}>
+            Set
+          </button>
         {/if}
       </div>
+    </div>
+    {#await Org.getOrgsByMember(profile.address, config)}
+      <Loading center />
+    {:then orgs}
+      {#if orgs.length > 0}
+        <div class="members">
+          {#each orgs as org}
+            <div class="member">
+              {#await Profile.get(org.address, ProfileType.Minimal, config)}
+                <Loading small margins />
+              {:then profile}
+                <div class="member-icon">
+                  <Link to="/orgs/{profile.address}">
+                    <Avatar source={profile.avatar ?? profile.address} address={profile.address} />
+                  </Link>
+                </div>
+                <div class="desktop">
+                  <Address address={profile.address} compact
+                    resolve noBadge noAvatar {profile} {config} />
+                </div>
+              {/await}
+            </div>
+          {/each}
+        </div>
+      {/if}
+    {:catch err}
+      <Message error>
+        <strong>Error: </strong> failed to load orgs: {err.message}.
+      </Message>
+    {/await}
+    <div class="projects">
+      {#if profile.anchorsAccount}
+        {#await Org.get(profile.anchorsAccount, config)}
+          <Loading center fadeIn />
+        {:then org}
+          {#if org}
+            {#await org.getProjects(config) then projects}
+              {#each projects as project}
+                <div class="project">
+                  <Project {project} user={addressOrName} config={profile.config(config)}>
+                    <span slot="stateHash">
+                      <span class="mobile">commit {formatCommit(project.anchor.stateHash)}</span>
+                      <span class="desktop">commit {project.anchor.stateHash}</span>
+                    </span>
+                  </Project>
+                </div>
+              {/each}
+            {:catch err}
+              <Message error>
+                <strong>Error: </strong> failed to load projects: {err.message}.
+              </Message>
+            {/await}
+          {/if}
+        {/await}
+      {/if}
+    </div>
```

Comment:
Here we had wad seems like a wrong indentation, feels great to find them myself :sweat_smile: